o1vm/src/interpreters/mips/tests_helpers.rs

```diff
@@ -89,6 +91,9 @@ where
         halt: false,
         // Keccak related
         syscall_env: SyscallEnv::default(),
+        selectors: Vec::from(std::array::from_fn::<Fp, N_MIPS_SEL_COLS, _>(|_| {
```

Comment:
It might be ok for a first version, but I would simply keep track of the actual integer representing the instruction, and in the `main.rs` file, I would build the actual vectors of evaluation points.